src/pages/Auth/Auth.module.scss

```diff
@@ -0,0 +1,61 @@
+* {
+	box-sizing: border-box;
+}
+
+.authorization {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+	height: 77vh;
+}
+
+.error {
+	color: red;
+	margin-bottom: 20px;
+}
+
+.form {
+	width: 270px;
+	padding: 10px 20px;
+	display: flex;
+	flex-direction: column;
+	gap: 18px;
+
+	.formLabel {
+		width: 100%;
+		height: 100%;
+	}
+
+	.formField,
+	.formBtn {
+		width: 100%;
+		padding: 10px 5px 10px 19px;
+		font-size: 15px;
+		line-height: 15px;
+		font-weight: 400;
+		border: none;
+		color: rgb(255, 255, 255, 0.7);
+		background-color: #9f9f9f;
+	}
+
+	.formField {
```

Comment:
make sure your teammates uses the same scss benefits in the same situations. your codestyle should align with everyone else